### ISSUE TITLE
VoiceOver doesn't allow small step sizes relative to large ranges

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#139]
From https://github.com/phetsims/sun/issues/495, we need to submit a bug report to Apple about this. If the `step` attribute for an input of type range is too small relative to the entire range of the input, the element won't receive alternative input.

**Comment by terracoda** [comment]
@jessegreenberg , is this issue about the custom small-step with the Shift key, or a control that has a small range and a default "smallish step size"?

**Comment by jessegreenberg** [comment]
This bug seems to happen when a slider has a very large range relative to a very small step size. We encountered this specifically in gravity-force-lab where the slider range spans 9 billion and the step size was 1. In this case, mobile VoiceOver failed to send DOM events to the browser when the user changed values with gestures. We fixed this in gravity-force-lab by making the step size 1 billion in this case.

Outside of a PhET simulation, a slider with a step size of 1 with a range of 9 billion would be very bad. But VoiceOver should still handle that case gracefully, so we would like to submit a bug report to Apple about it.

**Comment by jessegreenberg** [comment]
I just submitted a bug report to Apple, lets see if anything comes of it!

**Comment by jessegreenberg** [comment]
Apple responded by requesting we resubmit the report using Feedback Assistant and submitting any generated files that service provides.
